content/guides/users/create-app-user.md

```diff
@@ -42,6 +42,10 @@ be a name for the app user.
 To see all available optional parameters that may be set when creating an app
 user, see the [create user endpoint](endpoint://post-users).
 
+<Message type='notice'> Before you make any changes to the newly created
+account, you need to confirm the account creation through the link you
+receive in the confirmation email. </Message>
+
```

Comment:
Remove empty spaces before the first and the last word. Is there a way we could rephrase this sentence to a more simple one?